ci/new_tsqa/tests/test_spdy_protocol_select.py

```diff
@@ -0,0 +1,141 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+import requests
+import time
+import logging
+import subprocess
+
+import helpers
+
+import tsqa.test_cases
+import tsqa.utils
+import tsqa.endpoint
+
+log = logging.getLogger(__name__)
+
+class TestSPDY(helpers.EnvironmentCase):
+    environment_factory = {
+        'configure': {'enable-spdy': None},
+        'env': {'PKG_CONFIG_PATH': '/opt/spdylay/lib/pkgconfig/'},
```

Comment:
ah ok. this is the standard install path given at https://docs.trafficserver.apache.org/en/latest/admin/getting-started.en.html#building-traffic-server-with-spdy so i thought it would be ok but can totally swap for an environment variable.